faststream/rabbit/test.py

```diff
@@ -240,14 +240,19 @@ async def publish(
                     call = True
 
                 elif handler.exchange.type == ExchangeType.TOPIC:
-                    call = bool(
-                        re.match(
-                            handler.queue.routing_key.replace(".", r"\.").replace(
-                                "*", ".*"
-                            ),
-                            incoming.routing_key or "",
-                        )
-                    )
+                    call = True
+
+                    for current, base in zip_longest(
+                        (incoming.routing_key or "").split("."),
+                        handler.queue.routing.split("."),
+                        fillvalue=None,
+                    ):
+                        if base == "#":
+                            break
```

Comment:
@Lancetnik 

I think this needs to be a "continue" not a break.

The problem is that currently the routing_key `#.test.#` matches everything, but should only match messages containing "test", e.g. "test", "a.test", and "test.b".

Reply:
This is a common practice for all of **FastStream** cases. We are running just a first sutable handler. But, you are right, we can run all suitable once. Probably, I should refactor these cases, but I am too beasy with total project refactoring as u can see. Can you please create an Issue to add in backlog?